packages/react-server-gulp-module-tagger/index.js

```diff
@@ -22,12 +22,13 @@ module.exports = function(config) {
 	config.basePath = module.filename.replace(THIS_MODULE,'');
 	return forEach(function(stream, file){
 		return stream.pipe(replace(REPLACE_TOKEN, (match, optString) => {
+			const opts = optString ? {label: optString.replace(/.+label:\s?"(\w+)".+/, "$1")} : undefined;
```

Comment:
I added a test case for single quotes.  I'd be okay with postponing multiline options until we have more than one key supported by logger options (which might make it unreasonably difficult to maintain a reasonable maximum line width) or there's a concrete need for it -- what do you think?